site/zenodo_rdm/legacy/deserializers/schemas.py

```diff
@@ -38,6 +73,26 @@ def _creators(self, data):
             ret.append(r)
         return ret
 
+    def _funding(self, grants):
+        """Transform funding information."""
+        _funding = []
+        for grant in grants:
+            # format "10.13039/501100000780::190101904"
+            id_parts = grant["id"].split("::")
+            assert len(id_parts) == 2
+
+            # special case of funder loaded in Zenodo with no DOI related to it.
+            # it is part of the vocabulary.
+            funder_id = FUNDER_DOI_TO_ROR.get(id_parts[0], id_parts[0])
+
+            _funding.append(
+                {
+                    "funder": {"id": funder_id},
+                    "award": {"id": id_parts[1]},
```

Comment:
```suggestion
                    "award": {"id": id_parts[0 ] + "::" + id_parts[1]},
```

minor: I think `award.id`s are the fully formatted funder + code string (see example in [reference docs](https://inveniordm.docs.cern.ch/reference/metadata/#funding-references-0-n)).